dev-util/coccinelle/coccinelle-1.0.7.ebuild

```diff
@@ -0,0 +1,127 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python2_7 )
+
+inherit multilib eutils python-single-r1 bash-completion-r1 elisp-common autotools
+
+MY_P="${P/_/-}"
+DESCRIPTION="Program matching and transformation engine"
+HOMEPAGE="http://coccinelle.lip6.fr/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc emacs ocaml +ocamlopt pcre python test vim-syntax"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+# ocaml enables ocaml scripting (uses findlib)
+COMMON_DEPEND=">=dev-lang/ocaml-3.12:=[ocamlopt?]
+	dev-ml/sexplib:=[ocamlopt(+)?]
+	dev-ml/menhir:=[ocamlopt?]
+	dev-ml/camlp4:=[ocamlopt?]
+	dev-ml/parmap:=[ocamlopt?]
+	emacs? ( virtual/emacs )
+	ocaml? ( dev-ml/findlib:= )
```

Comment:
This doesnt have a subslot, you should remove `:=`

Reply:
Done.